migrations/1664278604452_add-table-to-record-flagged-posts.js

```diff
@@ -0,0 +1,27 @@
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+	pgm.createTable('post_flags', {
+		userId: {
+			type: 'integer',
+			notNull: true,
+			references: '"users"',
+			onDelete: 'cascade'
+		},
+		postId: {
+			type: 'integer',
+			notNull: true,
+			references: '"posts"',
+			onDelete: 'cascade'
+		},
+		reason: { type: 'text', notNull: false },
```

Comment:
`notNull` not needed if its not being set as true. This works anyway, same as without the `notNull` flag

Reply:
yes please

Reply:
i have pushed the changes